examples/docs_snippets/docs_snippets/guides/dagster/local_to_prod_tutorial/repository/repository_v4.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from dagster_snowflake import build_snowflake_io_manager
+from dagster_snowflake_pandas import SnowflakePandasTypeHandler
+
+from dagster import repository, with_resources
+
+from ..assets import comments, items, stories
+from ..resources.resources_v1 import hn_api_client
+
+# the snowflake io manager can be initialized to handle different data types
+# here we use the pandas type handler so we can store pandas DataFrames
+snowflake_io_manager = build_snowflake_io_manager([SnowflakePandasTypeHandler()])
+
+# start
+# repository.py
+
+
+@repository
+def repo():
+    resource_defs = {
+        "local": {
+            "hn_client": hn_api_client,
+            "io_manager": snowflake_io_manager.configured(
+                {
+                    "account": "abc1234.us-east-1",
+                    "user": "me@company.com",
+                    "password": "my_super_secret_password",
+                    "database": "SANDBOX",
+                    "schema": "ALICE",
+                }
+            ),
+        },
+        "production": {
+            "hn_client": hn_api_client,
+            "io_manager": snowflake_io_manager.configured(
+                {
+                    "account": "abc1234.us-east-1",
+                    "user": "dev@company.com",
+                    "password": "company_super_secret_password",
+                    "database": "PRODUCTION",
+                    "schema": "HACKER_NEWS",
+                }
+            ),
+        },
+    }
+    deployment_name = os.environ.get("DAGSTER_DEPLOYMENT", "local")
+
+    return [
+        with_resources(
```

Comment:
I realized that with this approach every job that is added to the repository will automatically have the snowflake IO manager if we're binding resources to jobs like this:
```
graph.to_job(resource_defs=resource_defs[deployment_name])
```
In my guide, I'll be adding other jobs to the repository and don't want them to use the snowflake IO manager by default. Thoughts on changing these assets to use [per-asset IO managers](https://docs.dagster.io/concepts/io-management/io-managers#per-asset-io-manager) and renaming the "io_manager" key to "snowflake_io_manager"?

Reply:
yeah i can definitely do that